docs/pages/blog/turbo-1-5-0.mdx

```diff
@@ -0,0 +1,148 @@
+- [**Prune now supported on pnpm, yarn and npm**](#prune-now-supported-on-npm-pnpm-and-yarn): Pruning your monorepo is now supported in all major package managers.
+- [**Improved environment variables in `turbo.json`**](#environment-variables-in-turbojson): Environment variables are now first-class citizens in your Turborepo pipeline configuration.
+- [**Changes to `package.json` hashing**](#changes-to-packagejson-hashing): We've improved how we hash `package.json` when running tasks.
+
+Update today by running `npm install turbo@latest`.
+
+## The Monorepo Handbook
+
+Setting up a monorepo for the first time often means navigating a lot of new concepts. You'll need to understand workspaces, package installation, sharing code and dependency management - and a lot more.
+
+This often meant that folks who wanted to set up a monorepo from scratch had to piece information together from different documentation sites. First `pnpm`, then `tsup`, then back to `changesets`, then back to Turborepo for dessert.
+
+We want to fill this gap with [the Monorepo Handbook](/docs/handbook). We've built guides on how to integrate all the tools you'll need to make ship happen with your monorepo, including guides on:
+
+- Installing Packages
+- Linting
+- Development Tasks
+- Building Apps
+- Publishing Packages
+
+{/* TODO - link these */}
+
+## `run` no longer needed
+
+You can now run tasks with the Turborepo CLI using `turbo <task>`.
+
+```diff
+- turbo run build
++ turbo build
+
+
+- turbo run lint build test
++ turbo lint build test
+```
+
+If your task name conflicts with a built-in `turbo` subcommand, we'll run our subcommand instead. That means you shouldn't name your tasks things like `prune`, `run`, or `login` - since those are built-in subcommands.
+
+`turbo run <task>` will continue to work, and there are no plans to deprecate it.
+
+## Prune now supported on `npm`, `pnpm` and `yarn`
+
+We're delighted to announce that `turbo prune --scope=<package-name>` is now supported in all major package managers.
+
+`turbo prune` lets you create a subset of your monorepo, focused on deploying a single application.
+
+Let's say that you have a monoreopo like this:
+
+```
+monorepo
+├─ apps
+│  ├─ api
+│  ├─ mobile
+├─ packages
+│  ├─ shared-utils
+│  ├─ tsconfig
+```
+
+If you want to deploy only `apps/api`, you can run:
+
+```bash
+turbo prune --scope="api"
+```
+
+You'd get an output that looks like this:
+
+```
+Generating pruned monorepo for api in /my-turborepo/out
+ - Added api
+ - Added shared-utils
+ - Added tsconfig
+```
+
+And the `/out` directory would look like this:
+
+```
+out
+├─ apps
+│  ├─ api
+├─ packages
+│  ├─ shared-utils
+│  ├─ tsconfig
+```
+
+Not only would `mobile` be removed, but **all of its dependencies would be removed from the root lockfile**. This means that you can create a streamlined version of your monorepo that only installs the dependencies it needs.
+
+### Deploying to Docker
```

Comment:
Add / recycle bits from docs, old post announcing this feature, and the docker example . This is actually more important and useful than the normal `prune` command (without docker)